app/js/tabmanager.js

```diff
@@ -225,9 +227,11 @@ const TabManager = {
    * @param {mixed} tabId
    *  Tab ID or Tab object.
    */
-  updateLastAccessed(tabOrTabId: chrome$Tab | number) {
+  updateLastAccessed(tabOrTabId: TabOrTabId | Array<TabOrTabId>) {
```

Comment:
I believe this should be the following:

    chrome$Tab | number | Array<chrome$Tab>

[`chrome.tabs.query`](https://developer.chrome.com/extensions/tabs#method-query) always passes an array of tabs to the `callback` param; the array won't ever contain IDs.

Reply:
While there is nothing at the moment that sends an array of IDs, it is supported by that code.

Reply:
Okay sure.